app/furniture/marketplace/cart/deliveries/_delivery.html.erb

```diff
@@ -1,13 +1,13 @@
-<%- if delivery.delivery_area.present? %>
-  <p class="text-lg font-bold"><%= render(delivery.delivery_area) %></p>
-<%- end %>
-<p class="text-left py-1 text-sm">
-  <%= render(Marketplace::Cart::DeliveryExpectationsComponent.new(cart: delivery.cart)) %>
-</p>
+
 <%= turbo_frame_tag(delivery) do %>
   <% if delivery.details_filled_in? %>
-    <p class="font-bold">Delivering to:</p>
-    <div class="rounded p-2 bg-orange-50">
+    <h3 class="font-bold">Delivering to:</h3>
+    <p>
+      <%= render(delivery.delivery_area) %><br />
+      <span class="text-sm font-light italic"><%= render(Marketplace::Cart::DeliveryExpectationsComponent.new(cart: delivery.cart)) %></span>
```

Comment:
Could this text styling become part of the component? Layout styling makes sense for wrapper tags but if these seem like inherent styling attributes of the component itself not of this "area" of the page.

Reply:
Sure, I think pushing this responsibility down is a good idea

Reply:
Ehh, actually let's not do that just yet. `DeliveryExpectationsComponent` is rendered in a number of places, with a substantially different information hierarchy. 

Presuming it should always be small, light, and italic does not seem to be valid.